lib/krikri/search_results_helper_behavior.rb

```diff
@@ -42,5 +42,51 @@ def render_bookmarks_control?
       false
     end
 
+    # Render enriched record for view
+    # @param [Krikri::SearchIndexDocument]
+    # @return [String]
+    def render_enriched_record(document)
+      agg = document.aggregation
+      return error_msg unless agg.present?
+      JSON.pretty_generate(agg.to_jsonld)
```

Comment:
I'm :+1: on this, but think it's outside the scope of the PR.  I think we'll want a `DPLA::MAP::Aggregation#to_map_json` that takes out the LDP data and squashes out superfluous bnode ids.

We also need to look at hosting our `@context` somewhere, so it doesn't need to appear in every serialized document.

I propose to consider this point dealt with for this PR by the creation of: https://www.pivotaltracker.com/story/show/87730010